lib/lint.ml

```diff
@@ -13,6 +13,33 @@ module Check = struct
   let marshal () = Yojson.Safe.to_string `Null
   let unmarshal _ = ()
 
+  let parse_errors_from_log job =
+    let job_id = Current.Job.id job in
+    let path = Current.Job.log_path job_id in
+    match path with
+    | Ok path ->
+      In_channel.with_open_text (Fpath.to_string path) (fun ic ->
```

Comment:
Sorry I didn't think of this before, but I'm wondering if we should be using `Lwt_io` functions here? I think these logs are usually pretty short, but in case they get a bit long, we may want to be sure that we will yield. WDYT?